organization/contact-details.md

```diff
@@ -23,8 +23,8 @@ We publish our news and announcements on:
 
 + [our mailing list](https://forms.gle/gn7wR2Eaxbv5g1BF9)
 + [our blog](https://blog.publiccode.net/)
-+ Twitter: @publiccodenet
++ Twitter
```

Comment:
I can't seem to find a reason for this in a commit message. However I don't know what value this adds.

Reply:
It's the second reference to Twitter on the page. If these were hyperlinks, I also wouldn't make every reference a hyperlink. But I'm open to the argument that not repeating the handle is unhelpful.

Reply:
I think that is a good argument!